cartography/intel/aws/route53.py

```diff
@@ -17,6 +18,75 @@
 aws_console_link = AWSLinker()
 
 
+@timeit
+def get_domains(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    domains = []
+    try:
+        client = boto3_session.client('route53domains', region_name=region)
+        paginator = client.get_paginator('list_domains')
+
+        page_iterator = paginator.paginate()
+        for page in page_iterator:
+            domains.extend(page.get('Domains', []))
+
+        for domain in domains:
+            domain['arn'] = domain['DomainName']
+            domain['region'] = region
+            domain['details'] = client.get_domain_detail(DomainName=domain['DomainName'])
+
+        return domains
+
+    except ClientError as e:
+        logger.error(f'Failed to call Route53Domains list_domains: {region} - {e}')
+        return domains
+
+
+def load_domains(session: neo4j.Session, domains: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    session.write_transaction(_load_domains_tx, domains, current_aws_account_id, aws_update_tag)
+
+
+@timeit
+def _load_domains_tx(tx: neo4j.Transaction, domains: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    query: str = """
+    UNWIND {Records} as record
+    MERGE (domain:AWSRoute53Domain{id: record.DomainName})
+    ON CREATE SET domain.firstseen = timestamp()
+    SET domain.lastupdated = {aws_update_tag},
+        domain.name = record.DomainName,
```

Comment:
let's create a property named `arn` here. So that it's standard across all the objects in cartography.